src/operations/extrusions/extrudeRectangularPath2.js

```diff
@@ -0,0 +1,35 @@
+const { path2 } = require('../../geometry')
+
+const { expand } = require('../expansions')
+
+const extrudeLinearGeom2 = require('./extrudeLinearGeom2')
+
+/**
+ * Expand and extrude the given geometry (path2).
+ * @param {Object} options - options for extrusion, if any
+ * @param {Number} [options.radius=1] - radius of the rectangle
+ * @param {Integer} [options.segments=0] - number of segments for rounded ends, or zero for chamfer
+ * @param {Array} [options.offset=[0,0,1]] - direction of the extrusion as a 3D vector
+ * @param {Number} [options.twistAngle=0] - final rotation (RADIANS) about the origin of the shape (if any)
+ * @param {Integer} [options.twistSteps=1] - number of twist segments about the axis (if any)
+ * @param {path2} geometry - the geometry to extrude
+ * @return {geom3} the extruded geometry
+ */
+const extrudeRectangularPath2 = (options, geometry) => {
```

Comment:
how does rectangular extrude of a path work ?

Reply:
I'm starting to think that extrudeRectangular() should be dropped. Yeah.

Basically, this was hacked functionality for...
- expanding a path to a geom2 (there are many ways)
- extruding the resulting geom2

There's really no reason for keeping this now. We have a very nice expand() function which can produce edges, chamfers, and rounds. We have a very nice linearExtrude() function. Why combine these into one ugly function?

Reply:
We want specialized API functions, which give users the building blocks. Why this nonsense?

@kaosat-dev thoughts?

Reply:
hmm very good question, there are extrudeRectangular functions (or similar) in all main CAd packages,  not sure
Let's leave it for now ? Or just have it be a function that wraps the functions you mention ?